data/CMakeLists.txt

```diff
@@ -15,29 +15,29 @@ else ()
   set(GTK_CSS_CLASS "active")
 endif ()
 configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/darktable.css.in ${CMAKE_CURRENT_BINARY_DIR}/darktable.css )
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/darktable.css DESTINATION ${SHARE_INSTALL}/darktable)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/darktable.css DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/darktable)
 
 #
 # web gallery export support files:
 #
 FILE(GLOB WEB_FILES "style/*")
-install(FILES ${WEB_FILES} DESTINATION ${SHARE_INSTALL}/darktable/style)
+install(FILES ${WEB_FILES} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/darktable/style)
 
 FILE(GLOB WEB_FILES "js/*")
-install(FILES ${WEB_FILES} DESTINATION ${SHARE_INSTALL}/darktable/js)
+install(FILES ${WEB_FILES} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/darktable/js)
 
 #
 # latex book template support files:
 #
 FILE(GLOB WEB_FILES "latex/*")
-install(FILES ${WEB_FILES} DESTINATION ${SHARE_INSTALL}/darktable/latex)
+install(FILES ${WEB_FILES} DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/darktable/latex)
 
 if(USE_LUA)
 #
 # lua system scripts
 #
-install(DIRECTORY "lua" DESTINATION ${SHARE_INSTALL}/darktable)
-install(FILES luarc DESTINATION ${SHARE_INSTALL}/darktable)
+install(DIRECTORY "lua" DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/darktable/)
```

Comment:
What's up with sometimes adding `/` and sometimes not?
I guess for consistency it is better to have no `/` at the end.